README.md

```diff
@@ -1,44 +1,42 @@
-# Redmadrobot Android KTX
- [![Build Status](https://img.shields.io/github/workflow/status/RedMadRobot/redmadrobot-android-ktx/CI/main?style=flat-square)][ci] [![License](https://img.shields.io/github/license/RedMadRobot/redmadrobot-android-ktx?style=flat-square)][license]
+# Gears :gear:
 
-Missing Android KTX extensions.
+[![Build Status](https://img.shields.io/github/actions/workflow/status/RedMadRobot/gears-android/main.yml?branch=main&style=flat-square)][ci]
+[![License](https://img.shields.io/github/license/RedMadRobot/gears-android?style=flat-square)][license]
```

Comment:
![image](https://github.com/RedMadRobot/gears-android/assets/18189767/5a569137-3daa-4fac-b523-fd8340801ffc)

The link works and if you go directly to the badge it's ok too - it seems to be ok afterwards

Reply:
Cache problem